packages/plugin-commands-installation/src/install.ts

```diff
@@ -124,6 +124,10 @@ export function help () {
             description: `The directory in which the ${WANTED_LOCKFILE} of the package will be created. Several projects may share a single lockfile`,
             name: '--lockfile-dir <dir>',
           },
+          {
+            description: 'Allows to install dependencies to a directory other than node_modules',
+            name: '--modules-dir <dir>',
+          },
           {
             description: 'Dependencies inside node_modules have access only to their listed dependencies',
```

Comment:
Since we now have the `modules-dir` option:

```js
description: 'Dependencies inside the modules directory will have access only to their listed dependencies',
```